### DISCUSSION TITLE
Modern​Flyouts original design

### DISCUSSION THREAD

**Discussion by chigau444** [ShankarBUS/ModernFlyouts#1111] [General]
### Description of the new feature / enhancement

Could you put an option to have this design? It's beautiful and it was the reason I downloaded the app.
Sorry if I make any mistake. It's my first time using github

### Scenario when this would be used?

![msedge_mzOkljPBEo](https://user-images.githubusercontent.com/110864314/183532061-ef195e31-0344-4d79-b8ad-a90c74e55ff0.png)

**Comment by Samuel12321** [comment]
what specifically do you like about the design on the website compared with 0.9.3?

**Reply by HeyItsJono** [reply]
They look similar to me, except the website one looks a lot more compact by virtue of not having the seeker bar present. Maybe an option to disable this would make the flyout look cleaner?

![image](https://user-images.githubusercontent.com/4620093/194995310-e7c9a207-5b27-4ce3-92e9-3ccf8549aa9c.png)

**Reply by HeyItsJono** [reply]
Actually, I've just realised there's a beta release. I agree with OP some aspects of the design were better in 0.93; there seems to be a new outline around the flyouts in 0.10 which doesn't look nearly as nice or clean as when it wasn't there. The new seekbar looks heaps better but the button to expand it upwards clutters the space a bit.

![image](https://user-images.githubusercontent.com/4620093/194997455-61662f48-8227-4995-8957-de5a9378742d.png)

**Comment by Samuel12321** [comment]
thanks for the feedback, should be improved in 0.11

**Reply by HeyItsJono** [reply]
Thank you!

**Reply by HeyItsJono** [reply]
Apologies for rebumping but if I could make one more suggestion on the topic, I was wondering if it would be possible to have varying shapes for the 'album art' of media?

So if I was watching a YT video, instead of the thumbnail being displayed with large black bars at the top/bottom to make it square, it would just show the thumbnail as a rectangle?

I think it would make ModernFlyouts look more consistent across different applications. Here's a comparison of current behaviour (top) vs a mockup of what I mean (bottom):

![image](https://user-images.githubusercontent.com/4620093/196067692-b3188145-d8df-45e0-8aa0-7499fe3c29b3.png)

![image](https://user-images.githubusercontent.com/4620093/196067791-b548805f-2c8b-4567-a975-60a387a555b7.png)

**Comment by ppfmagno** [comment]
I don't know if this is still in planning, but I do agree with @HeyItsJono ... maybe the outline should be an option...
And maybe the expand button could be more discrete...
Another thing I'd suggest is to make the seek bar, well, "seekable" even on the small state...
